src/skins/vector/css/matrix-react-sdk/views/rooms/RoomTile.css

```diff
@@ -61,21 +61,43 @@ limitations under the License.
 .collapsed .mx_RoomTile_badge {
     margin-top: -15px;
     right: 7px;
+    min-width: 12px;
+    height: 16px;
+    border-radius: 16px;
+    padding: 0px 4px 0px 4px;
+    z-index: 200;
+}
+
+.mx_RoomTile_highlight .mx_RoomTile_badge:after {
+    content: "";
+    position: absolute;
+    display: block;
+    width: 0;
+    height: 0;
+    margin-left: 7px;
+	border-top: 8px solid #ff0064;
```

Comment:
This is weird, I'm going to search the code base for tabs, as I'm definitely not putting then in, this code was copied from another section.